README.rst

```diff
@@ -0,0 +1,91 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-mlx90614/badge/?version=latest
+    :target: https://circuitpython.readthedocs.io/projects/mlx90614/en/latest/
+    :alt: Documentation Status
+
+.. image:: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+.. image:: https://travis-ci.org/mikeysklar/adafruit_CircuitPython_mlx90614.svg?branch=master
+    :target: https://travis-ci.org/mikeysklar/adafruit_CircuitPython_mlx90614
```

Comment:
These should be `adafruit/Adafruit_CircuitPython_MLX90614` instead of `mikeysklar/adafruit_CircuitPython_mlx90614`